CONTRIBUTING.md

```diff
@@ -41,7 +49,7 @@ For the static analysis, we use [`clippy`].
 
 ```sh
 # To format the source code in the entire repository
-cargo clippy --workspace
+cargo clippy --workspace --all-features --all-targets -- --warn clippy::cargo --allow clippy::multiple_crate_versions
```

Comment:
Is it intended that you forgot the `--deny warnings` here?

Reply:
Yep, this is intended as they will be outputted anyway.
But we can as well add it for all to clarify it (and unify even more).